packages/block-editor/src/store/selectors.js

```diff
@@ -2015,7 +2015,7 @@ export const getInserterItems = createSelector(
 	},
 	( state, rootClientId ) => [
 		state.blockListSettings[ rootClientId ],
-		state.blocks.byClientId,
+		state.blocks.byClientId.get( rootClientId ),
```

Comment:
This fixes an omission from #46204: that PR uses the `rootClientId` substate only in `canInsertBlockType`, but not in other selectors that call `canInsertBlockType` or `canIncludeBlockTypeInInserter` and therefore need to declare the same dependencies, too.